helpers.go

```diff
@@ -0,0 +1,67 @@
+/*
+ * Datadog API for Go
+ *
+ * Please see the included LICENSE file for licensing information.
+ *
+ * Copyright 2016 by authors and contributors.
+ */
+
+package datadog
+
+import "encoding/json"
+
+// Bool is a helper routine that allocates a new bool value
+// to store v and returns a pointer to it.
+func Bool(v bool) *bool { return &v }
```

Comment:
The `-m` flag to the rescue; it looks like the first: inline and straight to the heap:

```go
 go build -gcflags=-m helpers.go 
# command-line-arguments
<SNIP>
./helpers.go:15: can inline Bool
./helpers.go:15: &v escapes to heap
./helpers.go:15: moved to heap: v
<SNIP>
```